Assets/SteamVR_Unity_Toolkit/Scripts/Controls/2D/RadialMenu/VRTK_IndependentRadialMenuController.cs

```diff
@@ -0,0 +1,326 @@
+
+            Vector3 toController = controllerPosition - transform.position;
+            Vector3 projection = transform.position + Vector3.ProjectOnPlane(toController, transform.forward);
+
+            float angle = 0;
+            angle = Utilities.AngleSigned(transform.right * -1, projection - transform.position, transform.forward);
+
+            // Ensure angle is positive
+            if (angle < 0)
+            {
+                angle += 360.0f;
+            }
+
+            return angle;
+        }
+
+        private void ImmediatelyHideMenu(InteractableObjectEventArgs e)
+        {
+            ObjectUntouched(this, e);
+            if (disableCoroutine != null)
+            {
+                StopCoroutine(disableCoroutine);
+            }
+            SetColliderState(false, e); // Don't want to wait for this
+        }
+
+        private void SetColliderState(bool state, InteractableObjectEventArgs e)
+        {
+            if (addMenuCollider && menuCollider != null)
+            {
+                if (state)
+                {
+                    menuCollider.enabled = true;
+                    menuCollider.center = desiredColliderCenter;
+                }
+                else
+                {
+                    bool should = true;
+                    Collider[] colliders = eventsManager.GetComponents<Collider>();
+                    SphereCollider controllerCollider = e.interactingObject.GetComponent<SphereCollider>();
+                    foreach (Collider collider in colliders)
+                    {
+                        if (collider != menuCollider)
+                        {
+                            if (controllerCollider.bounds.Intersects(collider.bounds))
+                            {
+                                should = false;
+                            }
+                        }
+                    }
+
+                    if (should)
+                    {
+                        menuCollider.center = new Vector3(100000000.0f, 100000000.0f, 100000000.0f); // This needs to be done to get OnTriggerExit() to fire, unfortunately
+                        waitingToDisableCollider = true; // Need to give other things time to realize that they're not colliding with this anymore, so do it a couple FixedUpdates
+                    }
+                    else
+                    {
+                        menuCollider.enabled = false;
+                    }
+                }
+            }
+        }
+
+        IEnumerator DelayedSetColliderEnabled(bool enabled, float delay, InteractableObjectEventArgs e)
+        {
+            yield return new WaitForSeconds(delay);
+
+            SetColliderState(enabled, e);
+
+            StopCoroutine("delayedSetColliderEnabled");
+        }
+        #endregion Helpers
+
+        #region Unity Methods
+        private void Awake() {
+            menu = GetComponent<RadialMenu> ();
+        }
+
+        private void Start() {
```

Comment:
formatting of the brackets should be
```
{
}
```

(so separate lines)

Seems to be like this in a few places, worth looking through and fixing it :)

shame visual studio auto format can't be your friend :(